pom.xml

```diff
@@ -420,16 +419,13 @@
                 <module>spring-cloud-modules</module>
                 <!-- <module>spring-cloud-cli</module> --> <!-- Not a maven project -->
 
-                <module>spring-exceptions</module>
-                <module>spring-integration</module>
-                <module>spring-jenkins-pipeline</module>
-
                 <!-- <module>spring-roo</module> --> <!-- Not supported JAVA-17327 -->
 
-                <module>spring-security-modules/spring-security-ldap</module>
```

Comment:
this is needed, because spring-security-ldap was not upgraded to jdk17, so it is included in the default profile.